RestaurantReservation/Program.cs

```diff
@@ -1,2 +1,155 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using RestaurantReservation.Db.DataModels;
+using RestaurantReservation.Db;
+using RestaurantReservation.Db.Repositories;
+
+//CreateExampleCustomer();
+
+//Listmanagers();
+
+//GetReservationsByCustomer();
+
+//ListOrdersAndMenuItems();
+
+//ListOrderedMenuItems();
+
+//CalculateAverageOrderAmount();
+
+//GetReservationsByView();
+
+//GetEmployeesByView();
+
+CalculateTotalRevenue();
+
+GetCustomersWithPartySize();
+
+static void GetCustomersWithPartySize()
+{
+    var dbContext = new RestaurantReservationDbContext();
+    var customerService = new CustomerRepository(dbContext);
```

Comment:
the repository can internally handle the DbContext, keeping the calling code cleaner and ensuring separation of concerns.
  
 so this initiation can happen inside CustomerRepository in the constructor